src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * @author leslieharland
+ */
+
+package duke.storage;
+
+import duke.task.Deadline;
+import duke.task.EventObj;
+import duke.task.TaskList;
+import duke.task.Todo;
+import duke.task.SearchTask;
+import duke.task.Task;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+
+
+/**
+ * The Class Storage.
+ */
+public class Storage implements CheckTask {
+    public String filePath;
+
+    /**
+     * Instantiates a new storage.
+     *
+     * @param filePath the file path
+     */
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Write to file.
```

Comment:
Could the javadoc comment explain in more detail, what file is the method writing to?